include/install-vitasdk.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 get_download_link () {
-  curl -sL https://github.com/vitasdk/vita-headers/raw/master/.travis.d/last_built_toolchain.py | python - $@
+  wget -qO- https://github.com/vitasdk/vita-headers/raw/master/.travis.d/last_built_toolchain.py | python - $@
```

Comment:
now we don't need this change. isn't it?

Reply:
yes
However, It might be a good start to get ride of curl completly and use the same binary everywere (good to know: wget is natively available even in alpine container)